Lib/test/test_typing.py

```diff
@@ -1573,7 +1574,8 @@ class C(B[int]):
 
     def test_subscripted_generics_as_proxies(self):
         T = TypeVar('T')
-        class C(Generic[T]):
+        P = ParamSpec('T')
+        class C(Generic[T, P]):
```

Comment:
I'm confused. If C is generic over T and P, how is C[int] even valid? In general I'd rather not make this test dependent on ParamSpec -- I'd rather add a new test for that. Same below.